packages/applications/src/views/Home.jsx

```diff
@@ -0,0 +1,121 @@
+            <img alt='kernel fingerprint' src={linesVector} width={383} height={412} />
+          </div>
+          <div className='hidden lg:block lg:fixed lg:-top-12 lg:-right-52 lg:z-0'>
+            <img alt='kernel fingerprint' src={linesVector} width={442} height={476} />
+          </div>
+        </div>
+        <section className='absolute w-full'>
+          <div className='container mx-auto px-4'>
+            <div className='flex content-center items-center justify-center'>
+              <div className='w-full lg:w-3/4 px-4'>
+                <div className='relative flex flex-col min-w-0 break-words w-full lg:w-3/4 lg:mx-auto my-24'>
+                  <p className='text-center text-4xl md:text-6xl my-10'>
+                    You found us! Welcome.
+                  </p>
+                  <div className='text-lg'>
+                    <div key={homeState} className={`transition-all duration-400 ${fade ? 'opacity-0' : 'opacity-100'}`}>
+                      {homeState === HOMESTATE.INTRO && <AppIntro onLogin={handleLogin} />}
+                      {homeState === HOMESTATE.PROCESS && <Process />}
+                      {homeState === HOMESTATE.ASSURANCE && <Assurance />}
+                    </div>
+                    {/** Navigation Buttons */}
+                    <div className='flex flex-row flex-end justify-center align-center'>
+                      {homeState > HOMESTATE.INTRO &&
+                        <button
+                          className='bg-gray-900 text-white active:bg-gray-700 text-sm font-bold uppercase px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1'
+                          type='button'
+                          onClick={() => updateHomeState(homeState - 1)}
+                        >
+                          Back
+                        </button>}
+                      {homeState < HOMESTATE.ASSURANCE &&
+                        <button
+                          className='bg-gray-900 text-white active:bg-gray-700 text-sm font-bold uppercase px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1'
+                          type='button'
+                          onClick={() => updateHomeState(homeState + 1)}
+                        >
+                          {homeState === HOMESTATE.INTRO ? 'Begin' : 'Next'}
+                        </button>}
+                      {/** Create Your Keys CTA */}
+                      {
+                        homeState === HOMESTATE.ASSURANCE &&
+                          <button
+                            className='bg-gray-900 text-white active:bg-gray-700 text-sm font-bold uppercase px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1'
+                            onClick={handleLogin}
+                            type='button'
+                          >
+                            Create your keys
```

Comment:
My previous comment does not seem to have been addressed regarding returning users or am I missing something?